client/src/components/search/searchModule.tsx

```diff
@@ -0,0 +1,17 @@
+import { Beejs } from "../../process/Beejs";
+
+export function search(searchString: string) {
+  const bee = new Beejs();
+
+  return bee.getFeed().then((res) => {
+    const searchResults = {};
+
+    for (const key in res) {
```

Comment:
This branch is currently not working for me when I try to search, and this is where I'm hitting this error:
```
url.ts:46 Uncaught Error: URL is not valid!
    at wr (url.ts:46:1)
    at new Ar (bee.ts:115:1)
```

But rather than debug this, the approach should be to use Dapplets to index public archive content